<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>900</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Audio Control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>890</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>683</width>
         <height>880</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>645</width>
            <height>400</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>645</width>
            <height>430</height>
           </size>
          </property>
          <property name="title">
           <string>SPDIF</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="minimumSize">
              <size>
               <width>625</width>
               <height>310</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>625</width>
               <height>310</height>
              </size>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QwtPlot" name="qwtPlot_Zeitsignal">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>571</width>
                  <height>250</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>300</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>60</height>
              </size>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="maximumSize">
                 <size>
                  <width>130</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Auswahl Audio Device</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_Audio_In">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_Start">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>10</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>Start</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_Stop">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>Stop</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_8">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>33</height>
              </size>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Anzahl Kanaele</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_channels">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>20</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>I2C</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="5" column="0" colspan="4">
            <widget class="QGroupBox" name="groupBox_7">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLCDNumber" name="lcdNumber_Vcc">
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="frameShape">
                 <enum>QFrame::Box</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="lineWidth">
                 <number>1</number>
                </property>
                <property name="midLineWidth">
                 <number>0</number>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Flat</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Volt</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLCDNumber" name="lcdNumber_2_Ausgang">
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::Box</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Flat</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Volt</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0" rowspan="3" colspan="4">
            <widget class="QGroupBox" name="groupBox_6">
             <property name="minimumSize">
              <size>
               <width>303</width>
               <height>292</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>303</width>
               <height>292</height>
              </size>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="2" column="1">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Versorgungsspannung</string>
                </property>
               </widget>
              </item>
              <item row="2" column="4">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Mittelwert Ausgangssignal</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1" rowspan="3">
               <widget class="QwtThermo" name="Thermo_Vcc">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>250</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="4" column="4">
               <widget class="QwtThermo" name="Thermo_Ausgang">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>250</height>
                 </size>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="inputMethodHints">
                 <set>Qt::ImhNone</set>
                </property>
                <property name="borderWidth">
                 <number>2</number>
                </property>
                <property name="maxValue">
                 <double>100.000000000000000</double>
                </property>
                <property name="minValue">
                 <double>-100.000000000000000</double>
                </property>
                <property name="pipeWidth">
                 <number>10</number>
                </property>
                <property name="value">
                 <double>0.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="7">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Abfragerate</string>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="8">
            <widget class="QComboBox" name="comboBox_Abfragerate"/>
           </item>
           <item row="1" column="1">
            <widget class="QRadioButton" name="radioButton_Kanal1">
             <property name="text">
              <string>Kanal 1</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Kanalauswahl</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Auswahl COM Interface</string>
             </property>
            </widget>
           </item>
           <item row="2" column="5" rowspan="3" colspan="4">
            <widget class="QGroupBox" name="groupBox_5">
             <property name="minimumSize">
              <size>
               <width>255</width>
               <height>292</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>255</width>
               <height>292</height>
              </size>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="1" column="2" colspan="2">
               <widget class="QLCDNumber" name="lcdNumber_Temperatur">
                <property name="maximumSize">
                 <size>
                  <width>75</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Flat</enum>
                </property>
               </widget>
              </item>
              <item row="6" column="2">
               <widget class="QwtTextLabel" name="TextLabel"/>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_8">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string>Strom</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_13">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string>Volume</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLCDNumber" name="lcdNumber_Strom">
                <property name="maximumSize">
                 <size>
                  <width>75</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Flat</enum>
                </property>
               </widget>
              </item>
              <item row="3" column="2" colspan="2">
               <widget class="QLCDNumber" name="lcdNumber_Volume">
                <property name="maximumSize">
                 <size>
                  <width>75</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Flat</enum>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_9">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string>MUX</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_7">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string>Temperatur</string>
                </property>
               </widget>
              </item>
              <item row="4" column="2">
               <widget class="QLCDNumber" name="lcdNumber_MUX">
                <property name="maximumSize">
                 <size>
                  <width>75</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Flat</enum>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="label_10">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>75</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string>Fehlerstatus</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QRadioButton" name="radioButton_MUX1">
                <property name="text">
                 <string>MUX 1</string>
                </property>
               </widget>
              </item>
              <item row="7" column="2">
               <widget class="QRadioButton" name="radioButton_MUX2">
                <property name="text">
                 <string>MUX 2</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="5" column="5" colspan="4">
            <widget class="QSlider" name="horizontalSlider_Volume">
             <property name="maximum">
              <number>50</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="pageStep">
              <number>5</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksAbove</enum>
             </property>
             <property name="tickInterval">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2" colspan="3">
            <widget class="QComboBox" name="comboBox_COM"/>
           </item>
           <item row="1" column="3">
            <widget class="QRadioButton" name="radioButton_Kanal2">
             <property name="text">
              <string>Kanal 2</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
  <customwidget>
   <class>QwtThermo</class>
   <extends>QWidget</extends>
   <header>qwt_thermo.h</header>
  </customwidget>
  <customwidget>
   <class>QwtTextLabel</class>
   <extends>QFrame</extends>
   <header>qwt_text_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
